test/e2e/persistentips_test.go

```diff
@@ -116,7 +116,10 @@ var _ = DescribeTableSubtree("Persistent IPs", func(params testParams) {
 					WithPolling(time.Second).
 					ShouldNot(BeEmpty())
 
-				Expect(testenv.ThisVMI(vmi)()).Should(testenv.MatchIPsAtInterfaceByName(params.networkInterfaceName, Not(BeEmpty())))
+				Eventually(testenv.ThisVMI(vmi)).
+					WithPolling(time.Second).
+					WithTimeout(30 * time.Second).
+					Should(testenv.MatchIPsAtInterfaceByName(params.networkInterfaceName, Not(BeEmpty())))
```

Comment:
I wonder; while this seems OK, isn't it more conceptually correct to have something like a `ThisVMReadiness` wrapper that also checks if the interfaces have IPs ? Or decorate the expectation somehow ?